README.md

```diff
@@ -44,6 +44,9 @@ Find the installation instructions in the [official documentation](https://www.p
 
 Alternatively, you can [run Percona Backup for MongoDB as a Docker container](https://hub.docker.com/r/percona/percona-backup-mongodb).
 
+NOTE: The repo contains source code to build binaries. It is not a library and it is not intended to be used directly by calling exposed functions, types, etc.
```

Comment:
What's the purpose of this note? What message do we want to say, and in what context? Perhaps this should be moved to the very top?